sweetest/sweetest-api/build.gradle

```diff
@@ -15,15 +15,15 @@ apply from: '../publishing.gradle'
 setCustomArtifactName('sweetest')
 
 dependencies {
-    implementation fileTree(dir: 'libs', include: ['*.jar'])
+    api project(':sweetest:sweetest-base')
+    implementation project(':sweetest:sweetest-core')
+
+    implementation "io.cucumber:cucumber-junit:$versions.cucumber"
+    implementation "io.cucumber:cucumber-jvm:$versions.cucumber"
+    implementation "io.cucumber:cucumber-picocontainer:$versions.cucumber"
```

Comment:
This will get removed in a later PR, right?

Reply:
yes, for v2 it will. for API v1 it will stay to keep backwards-compatibility.